src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -21,13 +25,26 @@ const SubscriptionsUtilizedCard = lazy(() => import('../../SmartComponents/Subsc
 const PatchManagerCard = lazy(() => import('../../SmartComponents/PatchManager/PatchManagerCard'));
 const RemediationsCard = lazy(() => import('../../SmartComponents/Remediations/RemediationsCard'));
 
-const Dashboard = () => {
-
+const Dashboard = ({ workloads }) => {
     const permission = useContext(PermissionContext);
     const intl = useIntl();
+    const [supportsSap, setSupportsSap] = useState(true);
+
+    useEffect(() => {
+        const fetchSapSystems = async () => {
+            try {
+                const response = await API.get(SAP_FETCH_URL);
+                setSupportsSap(response.data.results[1].count > 0);
```

Comment:
I wouldn't rely on this approach as the API could change order
```suggestion
                setSupportsSap(response.data.results?.find(({ value }) => value )?.count > 0);
```

Reply:
yah yah good call, THOUGH one would think changing up that in the api would be ah breaking change 🦹‍♀️🦹‍♀️🦹‍♀️ 🦹‍♀️ grrrrrr